mantidproject/mantid
# Add Sample logs and Invalid values to training courses

**Description of work.**

In Mantid Basic Course I run through concepts
and using the Show Sample Logs interface and some filtering
In Python In Mantid I give some suggestions for accessing Sample Logs in a script.

**To test:**

Build docs-html and check that:
- Links work
- Explanations make sense!
- Images are clear and appear.
- Well formatted
- Code is useful
- Follows Docs guidlines: https://developer.mantidproject.org/Standards/DocumentationGuideForDevs.html#how-to-define-titles-sections-etc

Fixes #28568

*This does not require release notes* because **documentation**



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
